### PR TITLE
C1f3 crops testing

### PR DESCRIPTION
**Pull request by krb78**
This PR is adding the crop rotation jsons for c1f3 (Spruce Haven Farm) to the pilot testing branch 

I don't think there are other issues or PRs related to this. :) 

I don't think I need to update the change log. 

I've added crop rotation, fertilizer schedules, manure schedules, tillage schedules, the field specifications, and updated the metadata file "c1f3_wcrops_metatdata". 

There are two fields. One field starts in corn, the other field starts in alfalfa and they switch back and forth every 4 years between alfalfa and corn. 

Hopefully, this will work and give us our first real farm field level outputs. Fingers crossed it'll also give us some insight into how our purchased feed emission factors stack up to our field level emissions. We will also be able to see if we get reasonable yields and N&P losses from the fields.

### CODE REVIEW

File: input/data/crop/c1f3_alfalfa_corn_rotation.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
             "planting_skip": 0,
             "harvesting_skip": 0,
             "planting_days": [110],
-            "harvest_years": [2000, 2001, 2002, 2003, 2008, 2009, 2010, 2011, 2016, 2017, 2018, 2019],
+            "harvest_years": [2000, 2000, 2000, 2001, 2001, 2001, 2001, 2002, 2002, 2002, 2002, 2003, 2003, 2003, 2003, 2008, 2008, 2008, 2009, 2009, 2009, 2009, 2010, 2010, 2010, 2010, 2011, 2011, 2011, 2011, 2016, 2016, 2016, 2017, 2017, 2017, 2017, 2017, 2018, 2018, 2018, 2018, 2018, 2019, 2019, 2019, 2019],
             "harvest_days": [160, 190, 220, 140, 170, 210, 240, 140, 170, 210, 240, 140, 170, 210, 240, 140, 170, 210, 240],
             "harvest_operations": ["harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only","harvest_kill"],
```
Comment (Isgotkowitz) [inline input/data/crop/c1f3_alfalfa_corn_rotation.json]: This harvest schedule is saying that it is going to start harvesting alfalfa a bunch in the years 2000 through 2003, then leave it alive in the ground until 2008-2011, leave it alive again, then harvest it a bunch in 2016-2019 and then kill it.

File: input/data/crop/c1f3_alfalfa_corn_rotation_final.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            "harvest_days": [160, 190, 220, 140, 170, 210, 240, 140, 170, 210, 240, 140, 170, 210, 240, 160, 190, 220, 140, 170, 210, 240, 140, 170, 210, 240, 140, 170, 210, 240, 160, 190, 220, 140, 170, 210, 240, 140, 170, 210, 240,  140, 170, 210, 240],
+            "harvest_operations": ["harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_only", "harvest_kill"],
```
Comment (Isgotkowitz) [inline input/data/crop/c1f3_alfalfa_corn_rotation_final.json]: Same thing here as above.

File: input/data/crop/c1f3_alfalfa_corn_rotation.json
Comment (Isgotkowitz) [inline input/data/crop/c1f3_alfalfa_corn_rotation.json:1]: Two files that are present are `input/data/crop/c1f3_alfalfa_corn_rotation.json` and `input/data/crop/c1f3_alfalfa_corn_rotation_final.json`, I assume these are the same thing? If so, then only one of them should be kept and it's name should be `input/data/crop/c1f3_alfalfa_corn_rotation.json`.

File: input/data/crop/c1f3_corn_alfalfa_rotation_final.json
Comment (Isgotkowitz) [inline input/data/crop/c1f3_corn_alfalfa_rotation_final.json:1]: Same thing with this file name.

**Review by Isgotkowitz — CHANGES_REQUESTED** [review]
Noticed a couple things looking through these inputs. I think the alfalfa schedule is not set up to do what you want it to do. 
Also, I noticed a lot of files that had almost the same name, but one had "final" tacked on to the end. This is not a great practice, because it is easy to wind up with a lot of files that all have the same information but only is actually usable.

**Comment by krb78** [comment]
ok Ed, I think I addressed all of these :) see what you think!

**Review by jadamchick — APPROVED** [review]
I didn't try running it, but the input files look fine to me given Ed's requested changes. Hope it works!

**Comment by jadamchick** [comment]
Just noticing that on the pilot testing branch, there are 2 metadata files for c1f3 that seem to have crops -- "_metadata_wcrops.json" and "wcrops_metadata.json". Is one of them redundant/outdated and can be deleted?

**Comment by krb78** [comment]
c1f3_wcrops_metadata.json is the correct one. The other one can be deleted.

**Review by Isgotkowitz — APPROVED** [review]
I think these inputs look good, they appear to be running all the plantings and harvestings correctly based on the 2014-2022 simulation I looked at. I was trying to run the full 22 year simulation and I keep getting an error with the first alfalfa seeding, but I think that is a separate bug. I'll keep looking into it tomorrow.

**Pull request merged**